generic.go

```diff
@@ -736,3 +736,14 @@ func Split[V any](slice []V, numberOfGroups int) [][]V {
 	}
 	return result
 }
+
+// Take returns a slice with the specified number of items from `slice`.
+// You may also pass a negative integer to take the specified number of items from the end of the `slice`.
+func Take[V any](slice []V, n int) []V {
```

Comment:
We could have a `Reslice`, alias. WDYT?

Reply:
IMO `Take` isn't quite a `Reslice` because you can't specify both the beginning and the end